src/app/app.component.ts

```diff
@@ -14,6 +14,8 @@ import { Notifications } from '../providers/notifications';
   templateUrl: './app.html'
 })
 export class StockpileApp {
+  // Need to ViewChild Nav instead of using NavController because this is
```

Comment:
My mistake - when I was reading this, I thought these comments were function comments. Didn't realize they were inside a class.